staubli_val3_driver/val3/ros_server/start.pgx

```diff
@@ -22,7 +28,20 @@
   // See the License for the specific language governing permissions and
   // limitations under the License.
 
+  // Check for VAL 3 version >= 7.7
 
+  l_sVersionMajor = mid(getVersion("VAL3"),1,1)
+  l_sVersionMinor = mid(getVersion("VAL3"),2,1)
+  toNum(l_sVersionMajor, l_nVersionMajor, l_bOkMajor)
+  toNum(l_sVersionMinor, l_nVersionMinor, l_bOkMinor)
+  if(!l_bOkMajor or !l_bOkMinor)
+    popUpMsg("Error reading VAL 3 version")
+    taskKill("ros_server~")
+  endIf
+  if( !((l_nVersionMajor == 7) and (l_nVersionMinor >= 7)))
+    popUpMsg("Error: VAL 3 version is < 7.7")
```

Comment:
This may require some clearer documentation (in the readme or somewhere else).

It's a bit terse and I can imagine users not completely understanding *why* this would be a problem.